new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+#             Welcome   to   the   VIM   Tutor
+
+Vim is a very powerful text editor that has many commands. Unlike most text
+editors where you use the keyboard to enter text and arrow keys to move your
+cursor, in Vim you use the keyboard to enter commands to manipulate text.
+These commands allow you to express complex thoughts in just a few keystrokes
+to move through text and manipulate it much faster than in most text editors.
+There is a great amount of commands, too many to explain in a tutor such as
+this, but as you will see over the course of this tutorial series these
+commands are not arbitrarily chosen, they form a sort of language with its own
+grammar.
```

Comment:
Note: `there is a great amount of commands, too many to explain...` sounds a bit intimidating, perhaps rephrasing this would be a good idea. Maybe simply remove it and blend with the next paragraph:

>  These commands form a sort of language with its own grammar. This tutorial will help you learn it by introducing you to the set of commands we find is enough to use Vim as an all-purpose editor.